src/definitions/behaviors/form.js

```diff
@@ -1106,6 +1109,32 @@ $.fn.form = function(parameters) {
           asDirty: function() {
             module.set.defaults();
             module.set.dirty();
+          },
+          autoCheck: function() {
+            module.debug('Enabling auto check on required fields');
+            $field.each(function (_index, el) {
+              var
+                $el        = $(el),
+                $elGroup   = $(el).closest($group),
+                isCheckbox = ($el.filter(selector.checkbox).length > 0),
+                isRequired = $el.prop('required') || $elGroup.hasClass(className.required) || $elGroup.parent().hasClass(className.required),
+                isDisabled = $el.prop('disabled') || $elGroup.hasClass(className.disabled) || $elGroup.parent().hasClass(className.disabled),
+                validation = module.get.validation($el),
+                hasEmptyRule = validation
+                  ? (validation.rules.findIndex(function(rule) { return rule.type == "empty" }) > -1)
```

Comment:
`findIndex` is not supported in IE11 unfortunately 😢  You may use the `$.grep()` approach as you did in #1228

Reply:
@prudho reminder... 😉

Reply:
Yup, i'll try to fix this tomorrow.

Reply:
Done !

Reply:
Done !